pisek/config/task_config.py

```diff
@@ -369,9 +356,9 @@ def validate_globs(cls, value: list[str], info: ValidationInfo) -> list[str]:
         for glob in value:
             if glob == "@ith":
                 glob = f"{info.data['num']:02}*.in"
-            if not glob.endswith(".in"):
+            if not glob.endswith("*.in"):
```

Comment:
This is a pretty significant change, which will break old tasks. Is it really needed?

Reply:
You're right. Best to leave this as it.